### DISCUSSION TITLE
Using Fibers for managing requests

### DISCUSSION THREAD

**Discussion by DragK** [roadrunner-server/roadrunner#1079] [Ideas]
Hi, I'm curious if there are plans to add Fibers to manage multiple requests, or add integration with existing async projects like reactPHP or Amphp?

**Comment by rustatian** [comment]
@butschster

**Reply by rustatian** [reply]
We have this RFC: https://github.com/spiral/roadrunner-worker/issues/19
First of all, we need to update SF to `v3.0` with PHP 8.1 support.

**Reply by butschster** [reply]
Yeah, we are adding 8.1 support for spiral framework right now. I think in a couple of month we will add fibers for better performance.
